aws-py-hub-and-spoke-network/README.md

```diff
@@ -87,6 +87,14 @@ pulumi destroy
 pulumi stack rm
 ```
 
+## Troubleshooting
+
+You may encounter a condition where the security group fails to delete, this may be due to incomplete deletion of VPC endpoints. To fix this condition, perform the following in the AWS console:
```

Comment:
```suggestion
You may encounter a condition where the security group fails to delete. This may be due to incomplete deletion of VPC endpoints. To fix this condition, perform the following in the AWS console:
```